content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,186 @@
+dynamic infrastructure provisioning and advanced traffic routing.
+
+<!-- body -->
+
+## Design principles
+
+The following principles shaped the design and architecture of Gateway:
+
+* __Role-oriented:__ Gateway API kinds are modeled after organizational roles that are
+  responsible for managing Kubernetes service networking:
+  * __Infrastructure Provider:__ Manages infrastructure that allows multiple isolated clusters
+    to serve multiple tenants, e.g. a cloud provider.
+  * __Cluster Operator:__ Manages clusters and is typically concerned with policies, network
+    access, application permissions, etc.
+  * __Application Developer:__ Manages an application running in a cluster and is typically
+    concerned with application-level configuration and [Service][service] composition.
+* __Portable:__ Gateway API specifications are defined as [custom resources][crds] and are supported
+  by many [implementations][implementations].
+* __Expressive:__ Gateway API kinds support functionality for common traffic routing use cases
+  such as header-based matching, traffic weighting, and others that were only possible in [Ingress][ingress]
+  by using custom annotations.
+* __Extensible:__ Gateway allows for custom resources to be linked at various layers of the API.
+  This makes granular customization possible at the appropriate places within the API structure.
+
+Refer to the full Gateway API [documentation][gw-api] for additional details of these principles.
+
+## Resource model
+
+The Gateway API has three stable API kinds:
+
+* __GatewayClass:__ Defines a set of gateways with common configuration and managed by a controller
+  that implements the class.
+
+* __Gateway:__ Defines an instance of traffic handling infrastructure, such as cloud load balancer.
+
+* Route kinds: The Gateway API defines API kinds, HTTPRoute for example, that define protocol-specific
+  rules for mapping traffic from a Gateway listener to a representation of backend network endpoints.
+  These endpoints are often represented as a {{<glossary_tooltip text="Service" term_id="service">}}.
+
+The Gateway API is organized into different API kinds that have interdependent relationships to support
+the role-oriented nature of organizations. A Gateway object is associated with exactly one GatewayClass;
+the GatewayClass describes the gateway controller responsible for managing Gateways of this class.
+Route kinds such as HTTPRoute, are then associated to Gateways. A Gateway can filter the routes that
+may be attached to its `listeners`, forming a bidirection trust model with routes.
+
+The following figure illustrates the relationships of the three primary Gateway API resources:
+
+{{< figure src="/docs/images/gateway-kind-relationships.svg" alt="gateway-kind-relationships" class="diagram-medium" link="https://raw.githubusercontent.com/kubernetes/website/main/content/en/docs/images/gateway-kind-relationships.svg" >}}
```

Comment:
- Don't link to raw GitHub content like this; people might bookmark it and we don't want that.
- Please set alt attributes that are meaningful for people with accessibility needs